admin/app/components/solidus_admin/orders/show/address/component.html.erb

```diff
@@ -14,6 +14,7 @@
               <div class="relative">
                 <button
```

Comment:
No need to change it at this point, but did you consider a `<details><summary>` combo here?

Reply:
Yes, I had some problems trying to fit the address menu initially with `<details><summary>`.
Refactored ✔️  Let me know 👍